tests/js/web/NavMainComponentTest.js

```diff
@@ -570,8 +605,10 @@ describe('NavMainComponent', function () {
         });
 
         it('should hide the open tertiary or quaternary nav', function () {
-            expect(this.$navTertiary.hasClass('is-open')).to.be.false;
-            expect(this.$navQuaternary.hasClass('is-open')).to.be.false;
+            setTimeout(() => {
+                expect(this.$navTertiary.hasClass('is-open')).to.be.false;
+                expect(this.$navQuaternary.hasClass('is-open')).to.be.false;
+            }, 100);
```

Comment:
Return of the timeouts 😄 , if we're writing asynchronous tests, we need to use Mocha's `done` callback so that the assertion is run as part of the test: (https://mochajs.org/#detects-multiple-calls-to-done)